src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    public static final Map<String, Integer> fruits = new HashMap<>();
+
+    public static void add(String name, int quantity) {
+        if (Storage.fruits.containsKey(name)) {
+            quantity += Storage.fruits.get(name);
+        }
+        Storage.fruits.put(name, quantity);
+    }

```

Comment:
better to use exact logic in classes. because the process of adding for return and supply will be different in the future, and you have to delete this method, cause it will be unnecessary